examples/advanced/nlp-ner/utils/bert.py

```diff
@@ -0,0 +1,17 @@
+import torch.nn as nn
```

Comment:
Is this file duplicated in the custom folder?

Reply:
put it into test script to make it self-contained